src/js/pages/JobsPage.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import {RouteHandler} from 'react-router';
+
+import Page from '../components/Page';
+import TabsMixin from '../mixins/TabsMixin';
+
+var JobsPage = React.createClass({
```

Comment:
Sorry to do this, but we have a directive that new components should be classes extending `React.Component` (ref https://github.com/dcos/dcos-ui/pull/70#discussion-diff-61490798). Let's update this one.